fdbserver/TLogInterface.h

```diff
@@ -135,22 +135,6 @@ struct TLogConfirmRunningRequest {
 	}
 };
 
-struct VersionUpdateRef {
```

Comment:
Could you please add a comment before this new struct definition to explain why you added this and the performance gain you saw in the microbenchmark?

Reply:
There is no new struct definition in this PR, I am only removing this old `VersionUpdateRef` struct.